test/integration/mod/cabpump.mod

```diff
@@ -4,7 +4,8 @@ NEURON {
         SUFFIX cadyn
         USEION ca READ cai,ica WRITE cai 
         RANGE ca 
-	GLOBAL depth,cainf,taur 
+	GLOBAL depth,cainf,taur
+		RANGE var
      
```

Comment:
`nitpick` : can we fix the indent?

Reply:
I see that there are more issues with the indentation in this mod file. Will re-indent whole file separately.